components/browser/menu/src/main/java/mozilla/components/browser/menu/BrowserMenu.kt

```diff
@@ -29,7 +29,8 @@ import mozilla.components.support.ktx.android.view.isRTL
  * A popup menu composed of BrowserMenuItem objects.
  */
 class BrowserMenu internal constructor(
-    private val adapter: BrowserMenuAdapter
+    private val adapter: BrowserMenuAdapter,
+    private val extras: Map<String, Any>? = null
```

Comment:
I wonder why this is a cryptic `extras` map and not just a parameter for the desired width? 🤔

Reply:
Resolved with a separate param :)